base/tools/src/test/resources/certs/ca_signing.conf

```diff
@@ -0,0 +1,5 @@
+authorityInfoAccess = OCSP;URI:http://ocsp.pki.com/
```

Comment:
I'd suggest we use [example.com](https://en.wikipedia.org/wiki/Example.com) for all examples. Could you align the `=` too for consistency? Thanks.

Reply:
Updated!